sharedbuffers/mapped_struct.py

```diff
@@ -504,38 +506,38 @@ def get_wrapped_key(obj):
             return obj
         elif isinstance(obj, long):
             return obj & ~WRAP_MASK
-
-    if isinstance(obj, tuple):
-        return obj[1]
-    elif isinstance(obj, int):
-        return obj & ~WRAP_MASK
+    else:
+        if isinstance(obj, tuple):
+            return obj[1]
+        elif isinstance(obj, int):
+            return obj & ~WRAP_MASK
```

Comment:
You can simplify this.

The python2 and 3 sides are identical except for the `long` case.

``` python
if isinstance(obj, tuple):
    return obj[1]
elif python2:
    # py2 cases
else:
    # py3 cases
```

That way you avoid repeating the common case.